build-gradle-plugin/src/main/kotlin/cz/ackee/gradle/type/CustomBuildType.kt

```diff
@@ -0,0 +1,45 @@
+package cz.ackee.gradle.type
+
+sealed class CustomBuildType {
+
+    abstract val name: String
+    abstract val appIdSuffix: String?
+    abstract val appNameSuffix: String?
+    abstract val signingConfigName: String?
+    abstract val isMinifyEnabled: Boolean
+    abstract val isShrinkResources: Boolean
+    abstract val enableProguard: Boolean
```

Comment:
as we discussed, please make this a computed property from `isMinifyEnabled` and `isShrinkResources`

Reply:
Fixed: 078dd98